llama.cpp

```diff
@@ -806,6 +810,14 @@ struct llama_model_loader {
                         free(lt.data);
                     }
                     break;
+#elif defined(GGML_USE_KOMPUTE)
+                case GGML_BACKEND_GPU:
+                    lt.ggml_tensor->data = lt.data;
+                    ggml_vk_h2d_tensor(lctx.ctx_kompute, lt.ggml_tensor);
+                    if (!use_mmap) {
+                        free(lt.data);
+                    }
+                    break;
```

Comment:
I don't think you need this - you are already passing the "data" buffer later in the code